### PR TITLE
docs: Update roadmap.md

### PR DESCRIPTION
**Pull request by markgrover**
### Summary of Changes
Adding a few more items to the roadmap that have been brought up recently.

### Documentation
roadmap.md

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely, including a title prefix.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by feng-tao** [comment]
hey @markgrover could you share more context on the items? cc @danwom

File: docs/roadmap.md [hunk 1]
```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
...
 
 *Status*: planning
 
 ## Mid Term - Our Next steps
+#### Improve search ranking
+*What*: Update search ranking to be informed by "badges" that may exist on data sets e.g. deprecated, etc.
+
+*Status*: planning
+
+#### Notifications when a table evolves
```
Comment (feng-tao) [inline docs/roadmap.md:33]: we do have a notification system so far with jira / email. Could you share more on the difference?

**Comment by markgrover** [comment]
cc @danwom @feng-tao

File: docs/roadmap.md [hunk 2]
```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
...
+#### Integrate with Data Quality system
 *What*: Integrate with different data quality systems to provide quality score.
 
 *Status*: planning
 
 ## Mid Term - Our Next steps
+#### Improve search ranking
+*What*: Update search ranking to be informed by "badges" that may exist on data sets e.g. deprecated, etc.
+
+*Status*: planning
```
Comment (feng-tao) [inline docs/roadmap.md:31]: this one is committed for discovery and explore work.

File: docs/roadmap.md [hunk 3]
```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
...
+#### Notifications when a table evolves
+*What*: Notify users in Amundsen (akin to Facebook notifications or similar) when a table evolves. Owners of data and consumers of data will likely need to be notified of different things.
+
+*Status*: planning has not started 
+
+#### Commonly joined tables / browsing the data model
+*What*: As a data user, I would like to see commonly joined tables and how to join them.
+One option would be to show commonly joined tables and showing example join queries. Another option would be to provide a navigational experience for data model, showing foreign keys and which tables they come from.
+
+*Status*: planning has not started
```
Comment (feng-tao) [inline docs/roadmap.md:42]: nice

File: docs/roadmap.md [hunk 4]
```diff
@@ -19,12 +19,33 @@ The following roadmap gives an overview of what we are currently working on and
...
+#### Commonly joined tables / browsing the data model
+*What*: As a data user, I would like to see commonly joined tables and how to join them.
+One option would be to show commonly joined tables and showing example join queries. Another option would be to provide a navigational experience for data model, showing foreign keys and which tables they come from.
+
+*Status*: planning has not started 
+
+#### Curated navigation experience
+*What*: Currently Amundsen's experience is very focussed on search. However, especially for new users, an experience where they are able to navigate through the data hierarchy is very important. This item proposes to revamp the navigational experience in Amundsen (currently, barebones - based on tags) to do justice to the user need to browse through data sets when they don't know what to even search for.
+
+*Status*: planning
```
Comment (feng-tao) [inline docs/roadmap.md:47]: cc @danwom

File: docs/roadmap.md (hunk 1 above)
Comment (markgrover) [reply docs/roadmap.md:33]: This is around getting notifications in Amundsen itself. I remember Ryan (Lyft's intern) worked on this half.

User story here is that when a new column gets added in a bookmarked table, or if there's a delay in the table, the consumer would get notification.

File: docs/roadmap.md (hunk 2 above)
Comment (markgrover) [reply docs/roadmap.md:31]: If it's that close, then I may just take this out of the roadmap. In general, I do think there's a lot of scope for search improvements but it's not quite clear to me what we'd have to do for that.

Thoughts?

File: docs/roadmap.md (hunk 4 above)
Comment (markgrover) [reply docs/roadmap.md:47]: Yup, this is exactly the item @danwom has been thinking about, I just wanted an open source roadmap placeholder for it.

File: docs/roadmap.md (hunk 2 above)
Comment (feng-tao) [reply docs/roadmap.md:31]: agree, I think we haven't yet defined a concrete search pain point story which I recalled @danwom will do some user study as well?

**Comment by feng-tao** [comment]
@danwom could you take a look as well?

**Pull request merged**